src/ecalc/libraries/libecalc/common/libecalc/core/result/results.py

```diff
@@ -29,6 +32,56 @@ class CommonResultBase(EcalcResultBaseModel):
 class GenericComponentResult(CommonResultBase):
     id: str
 
+    @property
```

Comment:
Rename to `attributes` or sth? Make a separate type/class for this and make it possible to have "somehow" "(static) typing", e.g. check that the attribute actually exists?

Reply:
Isn't attributes more generic than columns? We want to indicate that this will give the attributes that are sequences. 

Are you thinking of a separate class that has the columns property? Then multiple inheritance or a subclass of GenericComponentResult? We have something similar to what I'm doing here in `dto.result.tabular_time_series.py`. TabularTimeSeries has a method to_dataframe method. What do you think of that solution?

Reply:
I think we should be able to have generic functions for merge and get_subset also, so a separate class for tabular_time_series with that logic makes sense to me. We are adding DataFrame functionality to the results, with a bit more control and some additional data. If I remember correctly that was also the case/reasoning when adding TabularTimeSeries to dto.result

Reply:
Ja, jeg skjønner. Det jeg tenkte å eksponere fra denne klassen var get_subset og merge, resten var hjelpemetoder. Så vi kan jo diskutere om det burde finnes en separat klasse for de to metodene eller om vi skal ha de på dto. Også kan vi se om vi skal flytte noe fra hjelpemetodene i egne klasser så klart.

Reply:
https://equinor-ecalc.atlassian.net/browse/ECALC-136